storage-proofs/src/circuit/drgporep.rs

```diff
@@ -718,26 +770,36 @@ mod tests {
         .expect("failed to synthesize circuit");
 
         assert_eq!(cs.num_inputs(), 18, "wrong number of inputs");
-        assert_eq!(cs.num_constraints(), 363392, "wrong number of constraints");
+        assert_eq!(cs.num_constraints(), 7683760, "wrong number of constraints");
```

Comment:
This is also a plausible culprit for a hypothetical timeout while generating groth parameters. @DrPeterVanNostrand

Reply:
(it was discussed in slack, but for posterity's sake) this is not why the tests are segfaulting, the tests for this PR are segfaulting because the circle ci instance is running out of RAM (not due to a circle ci timeout)